scripts/heat/2-instances-cinder.yaml

```diff
@@ -3,7 +3,7 @@ heat_template_version: 2013-05-23
 parameters:
   image:
     type: string
-    default: cirros-0.3.4-x86_64-tempest-machine
+    default: cirros-0.4.0-x86_64-tempest-machine
```

Comment:
Do we need to set default value when we're setting it explicitly always?

Reply:
No we don't need to. In fact we could even remove that default line entirely, thus ensuring we always fail with a "required parameter not supplied" instead of a bogus default one needs to track down the provenance of first...

+1ing anyway, though - the rest looks fine and I don't want to block the whole review on a philosophical nitpick. Feel free to remove the default or not.

Reply:
@jsuchome it just makes it convenient to manually use the heat stack for cloud8+, thats why I decided to update it.. (to be useable outside automation)